lib/models/__tests__/actor.test.js

```diff
@@ -0,0 +1,23 @@
+const Actor = require('../actor');
+
+describe('actors model', () => {
+  
+  it('validates all model properties', () => {
+    const data = {
+      name: 'Winona Ryder',
+      dob: 1971,
+      pob: 'Winona, Minnesota'
+    };
+
+    const actor = new Actor(data);
+    const errors = actor.validateSync();
+    expect(errors).toBeUndefined();
+
+    const json = actor.toJSON();
+
+    expect(json).toEqual({
+      ...data,
+      _id: expect.any(Object),
```

Comment:
you should also have tests in your models that check to make sure errors are thrown if the model is missing a required field